.github/workflows/dev-build-node-cpp-bindings.yml

```diff
@@ -256,7 +200,7 @@ jobs:
     - name: Build Binding
       run: |
         ./scripts/build-c-client.sh;
-        npm install;
+        npm ci;
```

Comment:
Is it possible to add `--ignore-scripts` here?

Reply:
Added --ignore-scripts and ran npm scripts manually without using npm run.